apps/coscrad-frontend/src/store/slices/resources/media-items/media-item-slice.ts

```diff
@@ -0,0 +1,19 @@
+import { IIndexQueryResult } from '@coscrad/api-interfaces';
+import { createSlice } from '@reduxjs/toolkit';
+import { buildInitialLoadableState } from '../../utils';
+import { buildReducersForThunk } from '../../utils/buildReducersForThunk';
+import { MEDIA_ITEMS } from './constants';
+import { fetchMediaItems } from './thunks';
+
+const initialState = buildInitialLoadableState<IIndexQueryResult>();
```

Comment:
Did we not need to include media items in the generic? Why does this work? Check type safety.

Reply:
Fixed. It would have been a problem when destructuring props in the hard-wired presenter.